src/NHibernate.Test/NHSpecificTest/GH3513/Fixture.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Linq;
+using NUnit.Framework;
+
+namespace NHibernate.Test.NHSpecificTest.GH3513
```

Comment:
It should be moved to `NHibernate.Test.NHSpecificTest.GH1313`. #1313 was a bug reported on our old Jira as NH-3513, which was using a NH prefix. We keep NH prefix for them, to avoid conflict with GitHub issues number, which we prefix with GH. So it could be `NHibernate.Test.NHSpecificTest.NH3513` too.

Reply:
Oh okay. I am going to fix that